datafusion/proto/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -232,11 +232,12 @@ async fn roundtrip_custom_listing_tables() -> Result<()> {
             WITH ORDER (c ASC)
             LOCATION '../core/tests/data/window_2.csv';";
 
-    let plan = ctx.sql(query).await?.into_optimized_plan()?;
```

Comment:
It always returned an [EmptyRelation](https://github.com/apache/arrow-datafusion/blob/main/datafusion/core/src/execution/context/mod.rs#L523) before, and did not achieve the testing purpose as expected.